packages/client/src/components/app/Layout.svelte

```diff
@@ -59,7 +59,7 @@
 
   export let logoPosition = "top" // "top" or "bottom"
   export let titleSize = "S"
-  export let titleColor // CSS color string, only affects title
+  export let color // CSS color string, only affects title
```

Comment:
I think this was an accidental change - this should remain as `titleColor`. Only the BBUI prop should be called `color`. If you wanted to rename this one you'd also need to update the navigation settings component which saves it, since it currently is still using `titleColor`, so right now this wouldn't actually work.

Best course of action - just renaming this back to `titleColor`, but keeping the BBUI Heading component using `color`.